### PR TITLE
Category Manager News Feeds count

### PR DESCRIPTION
**Pull request by pe7er**
This patch creates in "**Category Manager: News Feeds**" 4 new columns with the number of newsfeeds per category. It contains a helper file for com_newsfeeds that is used in com_categories to calculate the number of newsfeed items (published, unpublished, archived, and trashed) in each category.
## Test instructions:

The Category Manager: News Feeds (Components > NewsFeeds > Categories) does not display the number of News Feeds in each category.

screen shot 2015-05-13 at 15 17 09

Create some new newsfeeds that are unpublished, trashed & archived.
**Note** This patch **works only in combination with** #6916. Install both #6916 and this patch (the order of installation does not matter, you just need both).
After installing the patch, The Category Manager: NewsFeeds (Components > NewsFeeds > Categories) should displays 4 new columns.

screen shot 2015-05-13 at 15 17 09

Please also check this patch with the Hathor admin template.

### CODE REVIEW

File: administrator/components/com_newsfeeds/helpers/countitems.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		$query->join('LEFT', '#__newsfeeds AS ca ON ca.catid = a.id AND ca.published = 2');
+
+		// Count trashed items
+		$query->select('COUNT(DISTINCT ct.id) AS count_trashed');
+		$query->join('LEFT', '#__newsfeeds AS ct ON ct.catid = a.id AND ct.published = -2');
+
+		return $query;
+	}
+}
+
```
Comment (zero-24) [inline administrator/components/com_newsfeeds/helpers/countitems.php]: @pe7er 

Travis what you to remove the line 52 here :smile: 

```
FILE: ...oomla-cms/administrator/components/com_newsfeeds/helpers/countitems.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 52 | ERROR | Additional whitespace found at end of file
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------
```

**Comment by pe7er** [comment]
grrr... :smile: 
I've changed it and added the new PR.

**Comment by hans2103** [comment]
@test succes

Tested both desktop and mobile.

**Comment by woluweb** [comment]
nothing to mention, works fine<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6936">issues.joomla.org/joomla-cms/6936</a>.</sub>

**Comment by zero-24** [comment]
Thanks for testing. RTC'ing now!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6936">issues.joomla.org/joomla-cms/6936</a>.</sub>

**Comment by m-b-o** [comment]
works fine - a hover tooltip on the icons on top of table will be nice<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6936">issues.joomla.org/joomla-cms/6936</a>.</sub>

**Comment by Bakual** [comment]
Merged with https://github.com/joomla/joomla-cms/commit/a8e866f0f04c48fb2cc19e27d4b2b9306fd0ff00 into `3.5-dev`.

Thanks!

**Pull request closed without merging**